system_tests/transfer_test.go

```diff
@@ -36,3 +40,27 @@ func TestTransfer(t *testing.T) {
 		Fatal(t, "Unexpected recipient balance: ", bal2)
 	}
 }
+
+func TestP256Verify(t *testing.T) {
+	ctx, cancel := context.WithCancel(context.Background())
+	defer cancel()
+	builder := NewNodeBuilder(ctx).DefaultConfig(t, false)
+	builder.chainConfig.ArbitrumChainParams.InitialArbOSVersion = 30
```

Comment:
Could we also have a test that calling the precompile on ArbOS 20 doesn't return anything?

Reply:
Done.